dev_guide/managing_images.adoc

```diff
@@ -1089,16 +1090,48 @@ c8a648134623: Pushed
 latest: digest: sha256:be8bc4068b2f60cf274fc216e4caba6aa845fff5fa29139e6e7497bb57e48d67 size: 6273
 ----
 ====
+endif::openshift-enterprise[]
++
+ifdef::openshift-origin,openshift-dedicated[]
```

Comment:
According to Shawn, it looks like openshift-online needs to be included here too